app/Http/Controllers/QuestionApiController.php

```diff
@@ -32,4 +32,24 @@ public function delete(Question $question): JsonResponse
 
         return response()->json(['message' => 'Question deleted']);
     }
+
+    public function update(CreateQuestionRequest $request, Question $question): JsonResponse
```

Comment:
Since `CreateQuestionRequest` is now being used for creating and updating, I think it would be worth renaming it to just 'QuestionRequest' or something instead - seeing `CreateQuestionRequest` in `update` is kinda confusing.